gnovm/cmd/gno/bug_test.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "testing"
+
+func TestBugApp(t *testing.T) {
+	tc := []testMainCase{
+		{
+			args:        []string{"bug -h"},
+			errShouldBe: "flag: help requested",
+		},
+		{
+			args:        []string{"bug unknown"},
+			errShouldBe: "flag: help requested",
+		},
+	}
```

Comment:
It would be great to have at least one unit test that checks most of the things. Perhaps you can use regular expressions to verify the presence of important elements.

Reply:
@moul:

I don't want it to open browser every time someone runs tests. I'm still confused about whats the best way to write tests for this command. 

> Perhaps you can use regular expressions to verify the presence of important elements.

Sorry, I did not get this part. Can you explain a bit more?